docs/usage/making-your-attachments-interactive.md

```diff
@@ -12,7 +12,7 @@ Take a look at this response on Slack:
 This is how you would build that up
 
 ```php
-$this->respondToSlack()
```

Comment:
Instead of changing this in the docs, I'd rather have a PR that makes the current examples work.

Reply:
OK, please wait I will send new PR in few minutes.

Reply:
Here is the new PR https://github.com/spatie/laravel-slack-slash-command/pull/90